### ISSUE TITLE
[Bug]: Redis - Invalid argument during startup

### ISSUE THREAD

**Issue by ahajicek07** [beyondcode/herd-community#893]
### Platform

Windows

### Operating system version

Windows 10 Pro

### System architecture

Intel (x86)

### Herd Version

1.9.0

### PHP Version

PHP 8.3.8

### Bug description

Redis service won't work. immediately it says in the log `Invalid argument during startup: Failed to open the .conf file: Hajicek\.config\herd\config\services\e6c74a5e-1e02-463d-b36c-848d919264eb\redis.conf CWD=C:\Users\Alex Hajicek\.config\herd\bin\services\redis\7.0.0`

### Steps to reproduce

I literally just added the service and hit start. I've tried deleted the service and re-adding it.

### Relevant log output

```shell
Invalid argument during startup: Failed to open the .conf file: Hajicek\.config\herd\config\services\e6c74a5e-1e02-463d-b36c-848d919264eb\redis.conf CWD=C:\Users\Alex Hajicek\.config\herd\bin\services\redis\7.0.0
```

**Comment by sschlein** [comment]
Does the conf file exists at that location? If not, does it help if you create an empty one?

**Comment by ahajicek07** [comment]
The redis.conf exists
![image](https://github.com/user-attachments/assets/e7f9529e-6d0a-463a-8ac3-9438ff0711f7)

**Comment by ahajicek07** [comment]
Thank you so much, I really appreciate your support. Loving HERD so far.

**Comment by MilanDierick** [comment]
I'm affected by this as well, would appreciate a fix 😄

**Comment by sschlein** [comment]
I am sorry about that, can you try this pre-release version of Herd 1.9.1? [Download](https://herd-dev.fra1.digitaloceanspaces.com/test-builds/Herd-1.9.1-setup.zip)

**Comment by MilanDierick** [comment]
Yes! That did the trick, thanks!

**Comment by sschlein** [comment]
Fixed in 1.10.0
